src/components/layout/GenericLayout/Navigation/index.tsx

```diff
@@ -87,7 +87,6 @@ export const Navigation = styled.ol<Partial<CSS.Properties & { isActive: boolean
     display: flex;
     align-items: center;
     position: relative;
-    font-family: inherit;
```

Comment:
The font family isn't working in the hole explorer. I'll fix it in a new PR! Thanks